src/integrations/blocks/structured-data-blocks.php

```diff
@@ -157,7 +113,6 @@ public function enqueue_block_editor_assets() {
 			return;
 		}
 
-		$this->asset_manager->enqueue_script( 'structured-data-blocks' );
 		$this->asset_manager->enqueue_style( 'structured-data-blocks' );
```

Comment:
Perhaps we can go one step further and add the `editorStyle` attribute too?
Supposedly it can also handle the handles.

https://developer.wordpress.org/block-editor/reference-guides/block-api/block-metadata/#editor-style